api/MutationObserver.json

```diff
@@ -164,11 +164,11 @@
             ],
             "webview_android": [
               {
-                "version_added": true
+                "version_added": "≤37"
               },
               {
-                "version_added": true,
-                "version_removed": true,
+                "version_added": "≤37",
```

Comment:
I'm not a fan of these added+removed using the same value, but here both were true so it's still an improvement.